Tests/UITests/UITests.xcodeproj/project.pbxproj

```diff
@@ -580,7 +580,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "";
-				DEVELOPMENT_TEAM = 637867499T;
+				DEVELOPMENT_TEAM = "";
```

Comment:
Just double checking, was this intentional? Usually we keep our dev team within the project file?
See SpeziViews for example: https://github.com/StanfordSpezi/SpeziViews/blob/main/Tests/UITests/UITests.xcodeproj/project.pbxproj

Reply:
I removed these, didn't know our convention was to keep them. Will update.